resources/tickets.py

```diff
@@ -86,9 +87,24 @@ def get(self):
             return {"tickets": [ticket.json() for ticket in TicketModel.query.all()]}
 
     @pm_level_required
-    def post(self):
+    def put(self):
         data = Tickets.parser.parse_args()
         ticket = TicketModel(**data)
 
         ticket.save_to_db()
         return ticket.json(), 201
+
+    @pm_level_required
+    def post(self):
+        data = Tickets.parser.parse_args()
+
+        if not ("ids" in data and type(data["ids"]) is list):
+            return {"message": "Ticket IDs missing in request"}, 400
+
+        for id in data["ids"]:
+            ticket = TicketModel.find_by_id(id)
```

Comment:
find_by_id is deprecated. Please use the `find` method

Reply:
When doing a batch update:
 - How should we handle if one of the ids does not exist. Do we abort all, or do we update the ones that exist?

Reply:
I was going to bring up that batch update question on Wednesday 😄 As for the "find" vs "find_by_id", I see "find_by_id" being used in a bunch of places throughout the app, should I update all of them?